src/tools/kata-ctl/src/iptables.rs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2023 Alec Pemberton, Juanaiga Okugas
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use clap::{App, Arg, Parser, SubCommand, Command, Parser};
+use crate::args::{IptablesCommand};
+use reqwest::{Url};
+use std::{fs};
+use anyhow::Result;
+use shimclient::MgmtClient;
+use args::{Commands};
+use std::process::Command;
+use thiserror::Error;
+
+//kata-proxy management API endpoint, without code would not know the location of the unix sockets
+const DEFAULT_TIMEOUT: u64 = 30;
+const IP_TABLES_SOCKET: &str = "unix:///run/vc/sbs/{sandbox_id}/ip_tables";
+const IP6_TABLES_SOCKET: &str = "unix:///run/vc/sbs/{sandbox_id}/ip6_tables";
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error{
+    #[error("Invalid Container ID {0}")]
+    InvalidContainerID(String),
+}
+
+//Verify Id for validating sandboxID
+pub fn verify_id(id:&str) -> Result<(), Error>{
+    let mut chars = id.chars();
+
+    let valid = matches!(chars.next(), Some(first) if first.is_alphanumeric()
+    &&id.len() >1
+    && chars.all(|c| c.is_alphanumeric() || ['.', '-', '_'].contains(&c)));
+
+    match valid {
+        true => Ok(()),
+        false => Err(Error::InvalidContainerID(id.to_string())),
+    }
+}
+
+pub fn handle_iptables(_args: IptablesCommand) -> Result<()> {
+
+    //implement handle_iptables
+    // let args = KataCtlCli::parse();
+    // match args.command{
+    //     Commands::Iptables(args) => handle_iptables(args),
+    // }
+
+    let matches = Command::new("iptables")
+    .subcommand(Command::new("get"))
+    .subcommand(Command::new("set"))
+    .get_matches();
+
+    //checking for subcommand entered form user 
+    match matches.subcommand() {
+        Some(("get", get_matches)) => {
+            // retrieve the sandbox ID from the command line arguments
+            let sandbox_id = get_matches.value_of("sandbox-id").unwrap();
+            // check if ipv6 is requested
+            let is_ipv6 = get_matches.is_present("v6");
+            // verify the container ID before proceeding
+            verify_id(sandbox_id)?;//validate::verify_id(sandbox_id)
```

Comment:
Please can you remove all commented out code and redundant comments like this:

```suggestion
            verify_id(sandbox_id)?;
```